clang/tools/clang-linker-wrapper/LinkerWrapperOpts.td

```diff
@@ -75,7 +75,7 @@ def arch_EQ : Joined<["--"], "arch=">,
   Flags<[DeviceOnlyOption, HelpHidden]>, MetaVarName<"<arch>">,
   HelpText<"The device subarchitecture">;
 def triple_EQ : Joined<["--"], "triple=">,
-  Flags<[DeviceOnlyOption, HelpHidden]>, MetaVarName<"<triple>">,
+  Flags<[WrapperOnlyOption, DeviceOnlyOption, HelpHidden]>, MetaVarName<"<triple>">,
```

Comment:
Not really sure if we need this change. I will try to remove this after testing.

Thanks

Reply:
We seem to need this. Otherwise, I get the following error: /usr/bin/ld: unrecognized option '--triple=spir64'
Adding @mdtoguchi for comment.

Thanks

Reply:
Looks to be used as a filter - without it, it will pass options that are recognized in the table to be pushed along to the final host compilation.